src/app/pages/privacy-policy/privacy-policy.component.html

```diff
@@ -0,0 +1,45 @@
+<app-hill-background cover></app-hill-background>
+<div class="home-container">
+  <mat-card class="column">
+    <mat-card-title>
+      <h1 style="font-size:60px;" class="title">Privacy Policy</h1>
+    </mat-card-title>
+    <mat-card-content>
+        <h1>Collection of Information</h1>
+          <h2>Information you give</h2>
+            <p>Buzz About receives personal information when you log in using a Google account. This includes your email address.</p>
```

Comment:
Firebase Authentication may also collect a user's password, phone number, user agent, and IP address.

Reply:
We're more concerned with what information we have access to than what Firebase collects.